src/java/htsjdk/samtools/CRAMFileWriter.java

```diff
@@ -134,10 +149,6 @@ private void initCRAMWriter(final OutputStream indexOS, final ReferenceSource so
         setSortOrder(samFileHeader.getSortOrder(), preSorted);
         setHeader(samFileHeader);
 
-        if (this.source == null) {
-            this.source = new ReferenceSource(Defaults.REFERENCE_FASTA);
-        }
```

Comment:
If `Defaults.REFERENCE_FASTA` is set, is there any remaining mechanism for propagating it to `CRAMFileWriter`?